package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@reactivemarkets/platform-api",
+  "version": "0.0.0",
+  "description": "Generated api for Reactive Markets Platform",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "build/js/Message_generated.js",
```

Comment:
Tags will be taken from the "latest" branch. The generated code on the latest branch will be in `js`, not `build/js`.

Reply:
Fixed.